git_machete/git_operations.py

```diff
@@ -40,14 +40,14 @@ def __init__(self, cli_opts: CommandLineOptions) -> None:
         self.__contains_equivalent_tree_cached: Dict[Tuple[str, str], bool] = {}
 
     @staticmethod
```

Comment:
I'd also take a look at public attrs like `git_dir` while you're here, they can probably be made private as well